cmd/cmd.go

```diff
@@ -508,38 +488,18 @@ func generate(cmd *cobra.Command, model, prompt string, wordWrap bool) error {
 }
 
 func generateInteractive(cmd *cobra.Command, model string) error {
-	home, err := os.UserHomeDir()
-	if err != nil {
-		return err
-	}
+	/*
+		home, err := os.UserHomeDir()
+		if err != nil {
+			return err
+		}
+	*/
```

Comment:
```suggestion
```

Reply:
This is left over from the history stuff which will go back in soon.

Reply:
Ended up cutting this out and just sticking it directly inside of the new readline code.